src/app/components/top/solicitudes/solicitudes.html

```diff
@@ -19,9 +19,8 @@
                                     label="Hasta" class="fechas" [min]="fechaDesde">
                                 </plex-datetime>
                             </div>
-                            <div class="
-                                    col-3">
-                                <plex-text [(ngModel)]="paciente" name="paciente" (change)="filtrarPaciente()" label="Buscar paciente"></plex-text>
+                            <div class="col-3">
+                                <plex-text [(ngModel)]="paciente" name="paciente" (change)="onPacienteChange()" label="Buscar paciente"></plex-text>
```

Comment:
Agregale la propiedad debounce que te comente.
```suggestion
                                <plex-text [(ngModel)]="paciente" name="paciente" (change)="onPacienteChange()" label="Buscar paciente" debounce="400"></plex-text>
```